### DISCUSSION TITLE
Discussion on the README.md

### DISCUSSION THREAD

**Discussion by Agrover112** [EXYNOS-999/meta_resource#40] [Ideas]
Let's begin discussing about how we will write the final README.md which will transferred to our main Org -> UROPX

**Comment by VikramxD** [comment]
Lets first start with a format of how things would go

**Reply by Agrover112** [reply]
Sure  , I have few Qs , is this gonna be a README.md of entire meta?

**Reply by VikramxD** [reply]
of course

**Comment by Agrover112** [comment]
We will have to gather list of alll TOPICS whcihc we can put up on the main README.md using this for reference 👍https://github.com/vsouza/awesome-ios#readme

**Comment by rishistyping** [comment]
Higher priority right now is to to finish annotating and moving things from to_add.

**Reply by rishistyping** [reply]
If someone can finish the datasets in the submitted table and move them to the main table that'd be great help!
